app/services/appointments_manager/available_appointments.rb

```diff
@@ -9,7 +9,7 @@ def initialize(procedure, date)
 
   def call
     work_plan = doctor.work_plans.find_by!(day_of_week: date.wday)
-    appointments = doctor.appointments.filter_by_start_time(date).status_pending
+    appointments = doctor.appointments.filter_by_start_time(date).where(is_canceled=false)
```

Comment:
I think you can export no_canceled scope to appointment' model
```ruby
scope :not_canceled, -> { where(is_canceled: false) }
```
so it can be used like this
```suggestion
    appointments = doctor.appointments.filter_by_start_time(date).not_canceled
```